priv/repo/migrations/20151230080220_create_users.exs

```diff
@@ -0,0 +1,17 @@
+defmodule UeberauthProvider.Repo.Migrations.CreateUsers do
+  use Ecto.Migration
+
+  def change do
+    execute("CREATE EXTENSION citext;")
```

Comment:
This is just a little tester app. I'm assuming we're going to use it to explore and then throw it away